py3status/modules/whatismyip.py

```diff
@@ -22,12 +22,15 @@
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
     url_geo: IP to check for geo location (must output json)
-        (default 'https://freegeoip.net/json/')
+        (default 'https://ifconfig.co/json')
 
 Format placeholders:
     {icon} display the icon
-    {country} display the country
+    {country} display the country, eg: 'France'
+    {country_iso} display the country, eg: 'FR'
     {ip} display current ip address
+    {ip_decimal} display current ip address in decimal
+    {city} display the city
     any other key in JSON fetched from `url_geo`
```

Comment:
Missing `{hostname}`. Maybe kill `display the...` stuffs too.
```
    {icon}        eg ●, ■
    {country}     eg France
    {country_iso} eg FR
    {ip}          eg 123.45.67.890
    {ip_decimal}  eg 1234567890
    {city}        eg Paris
    {hostname}    eg py3.py3status.com
```

Reply:
Where {hostname} come from? From ipstack? I don't use it here. I don't put it because not in default url provider. Not in old url too.

Reply:
I missed this. `{hostname}` is in https://ifconfig.co/json here (6 keys). We good.